webviewer/src/pages/Signup/components/Card/ConfirmCard.jsx

```diff
@@ -3,9 +3,16 @@ import { Button } from 'components/_shared/Button';
 import { Header, CardText, CardWrap } from './Card.styles';
 import { useHistory } from 'react-router-dom';
 import { DEFAULT_ERROR, ROUTES } from '_constants';
+import { useUserActions } from 'state';
 
 export const ConfirmCard = ({ error }) => {
   const history = useHistory();
+  const { setAutoLogged } = useUserActions();
+
+  const handleContinueClick = async () => {
```

Comment:
and handleContinueClick name also encourage a little bit)

Reply:
changed